core/domain/user_query_jobs_one_off.py

```diff
@@ -102,3 +102,13 @@ def reduce(query_model_id, stringified_user_ids):
         user_ids = [ast.literal_eval(v) for v in stringified_user_ids]
         query_model.user_ids = [str(user_id) for user_id in user_ids]
         query_model.put()
+
+    @classmethod
+    def _post_completed_hook(cls, job_id):
+        # TODO: change query status in query model to completed.
```

Comment:
Actually, if the additional params method works, I'm fine with that too. So, on the whole, I'm fine with both options -- whatever works.